react-components/components.md

```diff
@@ -10,6 +10,17 @@ She knows that React will help her create the UI of her attendance webapp. But h
 
 This lesson will follow the story of Sofia, who will learn the syntax to create and use two components.
 
+### !callout-info
+
+## Sofia's React App
+
+We will use Sofia's web app throughout the Learn React lessons. There is a companion [Student App repository](https://github.com/AdaGold/student-app) with branches for each of the features added in the Learn lessons. This repository allows us to see all the parts working together. 
+</br>
+</br>
+There are a couple **Check For Understanding** prompts in the React lessons that ask us to recreate Sofia's app and add new features. We may complete these **Check For Understanding** prompts in a brand new React app or by using a fork of the [Student App repository](https://github.com/AdaGold/student-app) as a baseline.
```

Comment:
A few minor voicing changes
1. Can we commit to "a couple"? A like "a few" to hedge our bets
2. I prefer can over may. To me, I often expect "we may...." to be followed up with like an "or we may not." More of like a possibility where the alternatives are either doing a thing or not. Might just be me, but can sounds less loaded.
3. Suggest making it explicit that the new react app would be one the student makes. There's no alternative empty app starting repo, is there?
```suggestion
There are a few **Check For Understanding** prompts in the React lessons that ask us to recreate Sofia's app and add new features. We can either complete these **Check For Understanding** prompts in a brand new React app that we make ourself, or by using a fork of the [Student App repository](https://github.com/AdaGold/student-app) as a baseline.
```